cnf-certification-test/accesscontrol/rbac/automount.go

```diff
@@ -0,0 +1,49 @@
+package rbac
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	"github.com/test-network-function/cnf-certification-test/internal/clientsholder"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type AutomountToken struct {
+	serviceAccountName string
+	podNamespace       string
+	ClientHolder       *clientsholder.ClientsHolder
+}
+
+type ServiceAccountTokenStatus struct {
+	TokenSet   bool
```

Comment:
TokenSet   is really isAutomountServiceAccountTokenPresent
TokenValue is really automountServiceAccountTokenValue
I would just remove this object, see below

Reply:
Removed DetermineStatus().